font/ttf.ksy

```diff
@@ -242,6 +242,112 @@ types:
               - id: repeat_value
                 type: u1
                 if: repeat
+  gvar:
+    doc: >
+      gvar - Glyph Variations Table
+      This table provides all of the variation data that describe how TrueType glyph outlines in a 'glyf' table change across the font’s variation space.
+    seq:
+      - id: major_version
+        type: u2
+      - id: minor_version
+        type: u2
+      - id: axis_count
+        type: u2
+      - id: shared_tuple_count
+        type: u2
+      - id: shared_tuples_offset
+        type: u4
+      - id: glyph_count
+        type: u2
+      - id: flags
+        type: u2
+      - id: glyph_variation_data_array_offset
+        type: u4
+      - id: glyph_variation_data_offsets
+        type:
+          switch-on: flags & 0x01
+          cases:
+            0: u2
+            1: u4
+        repeat: expr
+        repeat-expr: glyph_count + 1
+    instances:
+      shared_tuples:
+        pos: shared_tuples_offset
+        type: tuple_record
+        repeat: expr
+        repeat-expr: shared_tuple_count
+      glyph_variation_data:
+        #The body of glyph_variation_datum should go here
+        #See https://github.com/kaitai-io/kaitai_struct/issues/436
+        type: glyph_variation_datum(_index)
+        repeat: expr
+        repeat-expr: glyph_count
+    types:
+      f2dot14:
+        seq:
+          - id: raw_coordinate
+            type: s2
+        instances:
+          coordinate:
+            value: raw_coordinate / 16384.0
+            -webide-parse-mode: eager
+        -webide-representation: "{coordinate:dec}"
+      tuple_record:
+        seq:
+          - id: coordinates
+            type: f2dot14
+            repeat: expr
+            repeat-expr: _parent.axis_count
+      tuple_variation_header:
+        seq:
+          - id: variation_data_size
+            type: u2
+          - id: tuple_index
```

Comment:
This should be a bit-type too.

Reply:
I'm not sure how to do that, there are 4 bits of flags and the 12 other bits are a number (this isn't just a bitfield). Those could be value instances I suppose. I'm not currently using the number here, but it should be used to point to the correct shared tuple when the peak tuple is implied.

Reply:
Changed to 4 bit flags and 12 bit index. Removed the bit masking in the process.